Upstream: Unknown
Reason: udev doesn't provide udevsettle symlink in newer versions
Source: Daniel Mierswa <impulze@impulze.org>

--- ./lib/libdevmapper.c.orig	2008-12-17 00:45:28.000000000 +0100
+++ ./lib/libdevmapper.c	2008-12-17 00:45:33.000000000 +0100
@@ -18,7 +18,7 @@

 #define	CRYPT_TARGET	"crypt"

-#define UDEVSETTLE	"/sbin/udevsettle"
+#define UDEVSETTLE	"/sbin/udevadm settle"

 static void run_udevsettle(void)
 {
